### DISCUSSION TITLE
USA Network Station call letters

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#42] [General]
I've been rather confused about how these type of stations should be entered into the database. For example, PBS stations most of the time have no call letters whatsoever (e.g. PBS Montana), while other stations from other networks (or countries for that matter), seem to use this convention exclusively. Would it be possible (obviously with effort and community support), to reformat the database so network stations are inserted into the database by using their proper call letters? Maybe it's the OCD ;) Thoughts?

**Comment by emveepee** [comment]
All OTA US stations including PBS will have call letters.  Rarely are these used in the OTA metadata, and people usually use their friendly name  however EPG sites like zap2it.com often use the call letters with a recognizable icon.

I don't know what database you are talking about, but the FCC maintains a call sign data linked to networks.

**Comment by AntiPontifex** [comment]
First of all, thanks for responding. The database I was talking about was the local database: I believe it is called channels.csv

Changing any/all OTA stations to just their call numbers would be, in my opinion, a good idea. As an option, the "friendly name" could be added as a secondary choice.

**Comment by emveepee** [comment]
Ok the best real source for the US is https://transition.fcc.gov/ftp/Bureaus/MB/Databases/cdbs/facility.zip the dat file is just pipe delimited text and it is updated each night,

**Comment by dtngfan85** [comment]
FYI, PBS Montana DOES have a callsign. Their flagship signal in Bozeman is KUSM-TV, while their repeater for the Missoula area is KUFM-TV.
